de_templates/ingest/Ingest_SourceType_SourceName/config/ingest_sources/ingest_config.json.jinja

```diff
@@ -1,7 +1,7 @@
 {% set pipeline_name = 'Ingest_' ~ dataset_name -%}
 {
     "data_source_name": "{{ pipeline_name }}",
-    "data_source_type": "{{ data_source_type }}",
+    "data_source_type": "{{ data_source_type.AZURE_SQL.value }}",
```

Comment:
That doesn't seem right to have this logic inside the template, I'll have a look whether we can do it in the CLI. Template should know nothing about the Enum.

Reply:
Fair, but I think it should be raised seperately in a bug ticket in that case. This PR is getting wildly out of hand imo

Reply:
I've proposed an update to this in the other comments, cause with explicit reference to AZURE_SQL this won't allow for any configuration. I think we'll need to replace Enum with Pydantic validator functions and use a plain string type for this field, cause Enum complicates it too much I guess. But we can do it in a follow up story. Can you please create one if you agree with changing the approach? :)

Reply:
Story for updating validation method: https://amido-dev.visualstudio.com/Amido-Stacks/_workitems/edit/6552

Reply:
This is done.